daemon/provider/musicvideoinfo/spotifyprovider.py

```diff
@@ -173,3 +174,46 @@ def get_tracks_by_album_id(self, album_id):
                 break
 
         return tracks
+
+	def getNewRelease(self, offset_count, limit):
+        self.logger.info("Get albums of the new release from Spotify.")
+        service_host = "https://api.spotify.com"
+
+        """
+            Get client_id and client_secret value from local.
+        """
+        config = configparser.ConfigParser()
```

Comment:
About config file, I think you should use the `config.py`.  You can refer the `xmusic.py`.